cypress/e2e/files/files-filtering.cy.ts

```diff
@@ -265,7 +265,7 @@ describe('files: Filter in files list', { testIsolation: true }, () => {
 
 		// go to other view
 		appNavigation.views()
-			.findByRole('link', { name: /Personal Files/i })
+			.findByRole('link', { name: /personal files/i })
```

Comment:
Just so greping the code with case sensitive enabled still finds it.

```suggestion
			.findByRole('link', { name: /Personal files/i })
```